src/main/java/com/conveyal/r5/profile/FastRaptorWorker.java

```diff
@@ -316,9 +316,16 @@ private void advanceScheduledSearchToPreviousMinute (int nextMinuteDepartureTime
 
             return result;
         } else {
-            // no frequencies, return result of scheduled search
-            if (saveAllStates) statesEachIteration.add(scheduleState[request.maxRides].deepCopy());
-            return new int[][] { scheduleState[request.maxRides].bestNonTransferTimes };
+            // No frequencies, return result of scheduled search, but duplicated by the number of
```

Comment:
I'm not sure it will be clear why accessibility would be artificially deflated if we didn't do this. Maybe just say something about how this avoids potential bugs where assumptions are made about how many results will be returned from a search, and mention this has already happened, and refer to #306 as an example.